Dockerfile.php5.6.busybox

```diff
@@ -1,21 +1,20 @@
 FROM ubuntu:xenial as builder
 
-ENV PHP_VERSION=5.6.31
-ENV PECL_MODULES imagick-3.4.3 redis-3.1.3 igbinary-2.0.1 memcached-2.2.0 ssh2-0.13 msgpack-0.5.7 runkit-1.0.4
 ENV VERSION=5.6
```

Comment:
We now only provide the major and minor version (5.6, 7.0, 7.1). The revision number is determined later by asking PHP.net in the build-php.sh script.